swap/src/env.rs

```diff
@@ -54,6 +56,8 @@ impl GetConfig for Mainnet {
             bitcoin_punish_timelock: PunishTimelock::new(144),
             bitcoin_network: bitcoin::Network::Bitcoin,
             monero_avg_block_time: 2.std_minutes(),
+            // We choose a timeout here that is a bit longer than the 10-confirmation lock for Monero transactions.
+            monero_lock_retry_timeout: 25.std_minutes(),
```

Comment:
we might need to revisit this and increase a bit, if there are "erroneous" early refunds. keeping in mind that a refund takes 10 minutes too

Reply:
25minutes is already very much on the safe side. This timeout is applied AFTER btc_tx_lock is confirmed. In most cases this shouldn't take more than a minute if the asb has enough unlocked funds.